kyaml/yaml/rnode.go

```diff
@@ -718,10 +718,10 @@ func (rn *RNode) MustString() string {
 
 // Content returns Node Content field.
 func (rn *RNode) Content() []*yaml.Node {
-	if rn == nil {
-		return nil
+	if yNode := rn.YNode(); yNode != nil {
+		return yNode.Content
```

Comment:
Same.
I think it's better to first store the value in the local variable of yNode and then check with if yNode == nil in an early-return format.